run.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python
+
+import os
+import time
+import cProfile
+import datetime
+
+from osmnames.init_database import init_database
+from osmnames.import_wikipedia import import_wikipedia
+from osmnames.import_osm import import_osm
+from osmnames.export_osmnames import export_osmnames
+
+
+while os.system("psql --username=postgres postgres -c 'select 1' > /dev/null 2>&1"):
+    print("waiting for postgresql")
+    time.sleep(2)
+
+
+profiler = cProfile.Profile()
+profiler.enable()
+
+init_database.run()
```

Comment:
Using the aliases and exposing the right functions from the modules, this could read much more pythonic:

```python
init_database()
import_wikipedia()
import_osm()
export_osmnames()
```